dasharo-compatibility/dcu.robot

```diff
@@ -82,7 +79,7 @@ DCU003.001 Change the bootsplash logo
     Should Contain    ${result.stdout}    Success
     Flash Firmware    ./dcu/coreboot.rom
 
-    Power On
```

Comment:
Will fail if Power On does not actually restart the device, the firmware will not be reloaded